ingest/bin/infer-dengue-serotype.py

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env python3
+
+import argparse
+import json
+from sys import stdin, stdout
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Dengue specific processing of metadata, infer serotype from virus_tax_id"
+    )
+    parser.add_argument(
+        "--virus_tax_id",
+        type=str,
+        default="virus_tax_id",
+        help="Column name containing the NCBI taxon id of the virus serotype, default: virus_tax_id",
+    )
+    parser.add_argument(
+        "--out_col",
```

Comment:
Ditto change flag to hyphen separated.

Reply:
Thanks! Fixed along with c9d1e2196503a16a63a0cb84be1caf2ac38a0ed7

Reply:
(I see this is updated in 1ff6af927bdefb290edf243dc9b21faaf644bb64)